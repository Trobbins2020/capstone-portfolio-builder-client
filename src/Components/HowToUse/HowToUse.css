.howtouse {
    margin: 10%;
    /* font-family: Arial, Helvetica, sans-serif; */
    letter-spacing: 1.5px;
}

.bold {
    font-weight: bolder;
}

.steps tr :first-child {
    font-weight: bolder;
}

.steps tr td {
    padding: 15px;
}