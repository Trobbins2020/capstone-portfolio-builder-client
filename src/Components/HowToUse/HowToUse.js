import React, { Component } from "react";
import Header from "../Header/Header";
import "./HowToUse.css";

export default class HowToUse extends Component {
  render() {
    return (
      <>
        <Header />
        <div className="howtouse">
          <h2 className="text-center">
            If you want to make your Portfolio for free, then you came at right
            place. you can generate your portfolio in few steps
          </h2>
          <br />
          <table className="steps">
            <thead>
              <tr>
                <td>Step I </td>
                <td>Register Your self</td>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>Step II </td>
                <td>Log In</td>
              </tr>
              <tr>
                <td>Step III </td>
                <td>
                  Choose Template and your will see a form that you need to fill
                </td>
              </tr>
              <tr>
                <td>Step IV </td>
                <td>
                  Download Template and that's sit you get your simple
                  portfolio.
                </td>
              </tr>
            </tbody>
            <tfoot></tfoot>
          </table>
        </div>
      </>
    );
  }
}
