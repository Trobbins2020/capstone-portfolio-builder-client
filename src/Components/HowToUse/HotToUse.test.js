import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import HowToUse from "./HowToUse";

describe(`HowToUse component`, () => {
  it("renders the Component", () => {
    const wrapper = shallow(<HowToUse />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
