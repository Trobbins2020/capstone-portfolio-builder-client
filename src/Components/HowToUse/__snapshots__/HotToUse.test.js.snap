// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowToUse component renders the Component 1`] = `
<Fragment>
  <Header />
  <div
    className="howtouse"
  >
    <h2
      className="text-center"
    >
      If you want to make your Portfolio for free, then you came at right place. you can generate your portfolio in few steps
    </h2>
    <br />
    <table
      className="steps"
    >
      <tr>
        <td>
          Step I 
        </td>
        <td>
          Register Your self
        </td>
      </tr>
      <tr>
        <td>
          Step II 
        </td>
        <td>
          Register Your self
        </td>
      </tr>
      <tr>
        <td>
          Step III 
        </td>
        <td>
          Choose Template and your will see a form that you need to fill
        </td>
      </tr>
      <tr>
        <td>
          Step IV 
        </td>
        <td>
          Download Template and that's sit you get your simple portfolio.
        </td>
      </tr>
    </table>
  </div>
</Fragment>
`;
